bedrock/base/templates/macros-protocol.html

```diff
@@ -227,23 +219,8 @@
 ) -%}
 <section class="mzp-c-card {% if class %}{{ class }}{% endif %} {% if aspect_ratio %}{{ aspect_ratio }}{% endif %} {% if media_icon %}{{ media_icon }}{% endif %} {% if youtube_id %}mzp-has-video has-video-embed{% endif %}" {% if attributes %}{{attributes|safe}}{% endif %}>
   <a class="mzp-c-card-block-link" href="{{ link_url }}" data-link-name="{{ ga_title }}" data-link-type="link" data-link-group="card" {% if tag_label %}data-card-tag="{{ tag_label }}"{% endif %}>
-    {% if image_url %}
-    <div class="mzp-c-card-media-wrapper">
-      {{ image(
-        alt=image_alt,
-        class='mzp-c-card-image',
-        height=image_height,
-        highres_image_url=highres_image_url,
-        include_highres=include_highres_image,
-        include_l10n=l10n_image,
-        loading='lazy',
-        sizes=image_sizes,
-        sources=image_sources,
-        srcset=image_srcset,
-        url=image_url,
-        width=image_width
-      ) }}
-    </div>
+    {% if image %}
+    <div class="mzp-c-card-media-wrapper">{{ image }}</div>
```

Comment:
Issue (blocking): since we're passing in an `<img>` as a string in `content_card()`, this needs the `safe` filter applied to avoid the markup being escaped:
```suggestion
    <div class="mzp-c-card-media-wrapper">{{ image|safe }}</div>
```